### PR TITLE
New package: wget2-1.99.2

### LINKED ISSUE #14254: Package request: wget2
- wget2 https://gitlab.com/gnuwget/wget2

1.99.2 (beta) -> https://gitlab.com/gnuwget/wget2/-/tags/1.99.2

GNU Wget2 is the successor of GNU Wget, a file and recursive website downloader.

### PR DESCRIPTION
**Pull request by CRKatri**
Closes #14254

### CODE REVIEW

**Comment by ericonr** [comment]
This is beta, from what I can see? https://lists.gnu.org/archive/html/bug-wget/2019-08/msg00029.html

Also, is there anything that'd make it impossible to update directly from our wget package? I think that's preferrable.

**Comment by CRKatri** [comment]
> Also, is there anything that'd make it impossible to update directly from our wget package?
wget2 is a successor of wget not a continuation, it uses the binary `wget2` not `wget`, so there is no way to update wget to wget2

File: srcpkgs/wget2/template [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+pkgname=wget2
+version=1.99.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-ssl=gnutls"
+hostmakedepends="pkg-config doxygen pandoc texinfo"
+makedepends="libidn-devel libuuid-devel pcre2-devel
+ nghttp2-devel gnutls-devel liblzma-devel libpsl-devel
+ brotli-devel libzstd-devel gpgme-devel libmicrohttpd-devel"
+depends="ca-certificates"
```
Comment (ericonr) [inline srcpkgs/wget2/template:11]: I wonder if it shouldn't be the library to depend on `ca-certificates`.

**Review by ericonr — COMMENTED** [review]
Have there been non-beta releases?

**Pull request closed without merging**